Dockerfile

```diff
@@ -44,3 +44,6 @@ LABEL io.k8s.description="driver-toolkit is a container with the kernel packages
       name="driver-toolkit" \
       io.openshift.release.operator=true \
       version="0.1"
+
+# Last layer for mapping the driver-toolkit to the corresponding Node     
+RUN  echo "{ \"KERNEL_VERSION\": \"${KERNEL_VERSION}\", \"RT_KERNEL_VERSION\": \"${RT_KERNEL_VERSION}\", \"RHEL_VERSION\": \"${RHEL_VERSION}\" }" > /etc/driver-toolkit-release.json
```

Comment:
The `KERNEL_VERSION` and `RT_KERNEL_VERSION` are not currently set in the ART builds. They are just enabling the right repositories to ensure the correct versions are installed. For example the downstream reconciled Dockerfile is setting only RHEL_VERSION: https://pkgs.devel.redhat.com/cgit/containers/driver-toolkit/tree/Dockerfile?id=4e7257855176edb2818cb54f7aae09415ba88d06.

As @zvonkok mentioned on Slack, we can use `rpm -qv` or something like that to set these variables to the installed versions.

Reply:
Thanks @dagrayvid I think this should work: 

```
rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}\n"  kernel-core
```

Reply:
Yes, looks good inside the driver-toolkit:
```
[root@6df1765fc4c5 kmods-via-containers]# rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}\n"  kernel-core
4.18.0-240.22.1.el8_3.x86_64
[root@6df1765fc4c5 kmods-via-containers]# rpm -q --qf "%{VERSION}-%{RELEASE}.%{ARCH}\n"  kernel-rt-core
4.18.0-240.22.1.rt7.77.el8_3.x86_64
```

Reply:
Update PTAL